python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -52,6 +54,7 @@ def __init__(
         load_options: LoadOptions | None = None,
         columns_names_capitalization: ColumnCapitalization = "original",
         enable_native_fallback: bool | None = LOAD_FILE_ENABLE_NATIVE_FALLBACK,
+        pandas_options: PandasLoadOptions | None = None,
```

Comment:
Would it make sense for us to deprecate the `ndjson_normalize_sep` and have it as part of  `PandasLoadOptions`?

Reply:
+1

Reply:
@tatiana Deprecation make sense but would be better to keep it in separate PR tracking here https://github.com/astronomer/astro-sdk/issues/1518